cloudflare.php

```diff
@@ -316,68 +148,7 @@ function match_domain_to_zone($domain, $zones)
     return;
```

Comment:
This is also useless. I just realized this was here. I could have used it in subdomain check code 😄 It's also buggy. It doesn't work with `domain.co.uk` 😄